### PR TITLE
Use the show enrollment page to show workshop details from the landin…

### PR DESCRIPTION
**Pull request by mehalshah**
…g page

### CODE REVIEW

**Comment by mehalshah** [comment]
image

**Comment by mehalshah** [comment]
Workshop details HAML page

image

File: dashboard/app/views/pd/workshop_enrollment/show.html.haml [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
   Thank you for registering.
 
-%p
-  You will receive a confirmation email.
+- if params.has_key?(:details)
+  = render partial: 'workshop_details'
+- else
+  %p
+    You will receive a confirmation email.
+
```
Comment (aoby) [inline dashboard/app/views/pd/workshop_enrollment/show.html.haml]: This seems like odd logic for this view. What about a different route? Your usage actually seems more appropriate for `show`. Maybe the "Thank you for registering" and "You will receive a confirmation email" parts should only be present here with a param set that happens immediately after registration, or these should be separated into two views. Thoughts?

Comment (mehalshah) [reply dashboard/app/views/pd/workshop_enrollment/show.html.haml]: Hmm - yeah, separate views seems better.

**Pull request merged**